__docs__/wonder-blocks-typography/heading.stories.tsx

```diff
@@ -134,6 +134,9 @@ export const SizesAndWeights = {
                 <Heading size="small" weight="semi">
                     Small semibold
                 </Heading>
+                <Heading size="small" weight="medium">
```

Comment:
I think the `AllVariants` component would be helpful in this example to show all the different size and weight combinations! (same for BodyText!)

Reply:
If it's alright, I'll defer that work to https://khanacademy.atlassian.net/browse/WB-1957!